privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -322,6 +322,20 @@
         </div>
     </div>
 
+    <div class="form-group">
+        <label for="time" class="col-sm-3 control-label"
+               translate>Priority</label>
+
+        <div class="col-sm-9">
+            <input type="number" class="form-control"
+                   max="999" min="1"
+                   ng-model="params.priority" />
+            <p class="help-block" translate>
+                In case of conflicting policies, the policy with the lowest priority number will take precedence.
+            </p>
+        </div>
+    </div>
+
```

Comment:
In the event handler list we also have an input field to change the "ordering" (priority) of the event handlers, which is actually used to determine, in which order the event handlers are applied.
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/static/components/config/views/config.events.list.html#L19

I would love to be able to also change the same/similar way the priority in the policy list. This way the admin could "sort" the policy list and move policies up and down.